DSpace/dspace-angular
# Meta "generator" tag in page header

## References
* Fixes https://github.com/DSpace/dspace-angular/issues/747

## Description
This PR ensures that a "Generator" meta tag is set in the page header of each page in the DSpace UI.

## Instructions for Reviewers

List of changes in this PR:
* Ensures that a "Generator" meta tag is set in the page header of each page within DSpace
* Aded tests to verify this

How to test this PR:
* Setup this branch with a REST api that supports showing the dspaceVersion on the root endpoint
* Inspect the DSpace UI page and verify that a "Generator" meta tag is present in the page headers

## Checklist
